tests/bluetooth/bsim_bt/bsim_test_adv_chain/prj.conf

```diff
@@ -5,6 +5,11 @@ CONFIG_BT_EXT_ADV=y
 CONFIG_BT_EXT_ADV_MAX_ADV_SET=2
 CONFIG_BT_DEVICE_NAME="Broadcaster Multiple"
 
+# Enable the experimental Advertising Data chaining in Zephyr Bluetooth LE
```

Comment:
```suggestion
# Enable the Advertising Data chaining in Zephyr Bluetooth LE
```

Suggest to remove experimental so that we don't need to remember to modify this comment once "[EXPERIMENTAL]" is removed.

Reply:
done